scripts/flow.sh

```diff
@@ -124,6 +124,11 @@ if [ -z $LAYER0_PREFIX ]; then
     exit 1
 fi
 
+if [ -z $LAYER0_API_ENDPOINT ]; then
```

Comment:
where is this used?

Reply:
It needed it to do the delete flow. Or is flow only for local use? In which case I just needed the api running locally.

Reply:
It can be used for either

Reply:
It's your call then to add this or not.

Reply:
I'm going to not add this because I think it's more useful to have both options available.